crates/api/src/runtime.rs

```diff
@@ -227,11 +249,51 @@ impl Default for Config {
     }
 }
 
+#[cfg(feature = "arbitrary")]
+impl Arbitrary for Config {
+    fn arbitrary(u: &mut Unstructured<'_>) -> arbitrary::Result<Self> {
+        let (
+            debug_info,
+            wasm_threads,
+            wasm_reference_types,
+            wasm_simd,
+            wasm_bulk_memory,
+            wasm_multi_value,
+            strategy,
+            cranelift_debug_verifier,
+            cranelift_opt_level
+        ) = <(bool, bool, bool, bool, bool, bool, Strategy, bool, OptLevel) as Arbitrary>::arbitrary(u)?;
```

Comment:
Out of curiosity, would this perhaps be more readable as:

```rust
let mut config = Config::new();
config.debug_info(Arbitrary::arbitrary(u)?)
    .wasm_threads(Arbitrary::arbitrary(u)?)
    .wasm_reference_types(Arbitrary::arbitrary(u)?)
    /* ... *?
```

Reply:
I originally did this so that the `size_hint` would be easier to follow, but I've revamped this PR now so that this isn't an issue.